WiserTaskScheduler/WiserTaskScheduler/Core/Workers/ConfigurationsWorker.cs

```diff
@@ -15,6 +15,7 @@ public class ConfigurationsWorker : BaseWorker
         private readonly IConfigurationsService configurationsService;
         
         public ConfigurationModel Configuration { get; private set; }
+        public bool HasAction => configurationsService.HasAction;
```

Comment:
Is het 100% zeker dat `configurationsService` nooit `null` kan zijn hier? Zo nee, dan graag een controle toevoegen daarop.